kubetest/e2e.go

```diff
@@ -37,6 +37,9 @@ func run(deploy deployer, o options) error {
 	os.Setenv("KUBE_CONFIG_FILE", "config-test.sh")
 	// force having batch/v2alpha1 always on for e2e tests
 	os.Setenv("KUBE_RUNTIME_CONFIG", "batch/v2alpha1=true")
+	// force having apps/v1beta2 always on for e2e tests
+	// TODO: remove it once we enable apps/v1beta2 by default
+	os.Setenv("KUBE_RUNTIME_CONFIG", "apps/v1beta2=true")
```

Comment:
@fejta Referring [this](https://github.com/kubernetes/kubernetes/blob/master/cluster/vagrant/config-default.sh#L96-L97),  the environment variable triggers behaviors of inputted version.

Currently, `v1beta2` is disabled by default. We need to turn on `v1beta2` behaviors for testing purpose during implementation stage.